### ISSUE TITLE
[BUG] Some commands passed to Invoke-PnPSPRestMethod don't return usable objects

### ISSUE THREAD

**Issue by joshtransient** [pnp/powershell#188]
### Reporting an Issue or Missing Feature
Occasionally, Invoke-PnPSPRestMethod will return an object that has data in its `value` property, but is not accessible. Haven't determined a pattern yet, but so far it is limited to calls that return more than one object.

Output usually looks like this in the legacy module (truncated):
```
Key   Value
---   -----
value {System.Collections.Generic....
```

In PnP.PowerShell, I see (truncated):
```
Name    Value
----    -----
value   [{"LoginName":"c:0....
```

### Expected behavior 
Use Invoke-PnPSPRestMethod to get an object I can use in a script. For multi-valued results, use `.value` to access those results, e.g.
```
$groups = Invoke-PnPSPRestMethod -Method get -Url "/_api/web/SiteUsers?`$filter=PrincipalType eq 4&`$select=Title,LoginName"`
$groups.value
```
<returns the value(s)>

### Actual behavior 
`$groups.value` returns:
```
ValueKind
---------
    Array
```

### Steps to reproduce behavior
Run the example from the expected behavior section

### What is the version of the Cmdlet module you are running?
PnP.PowerShell 1.2.0
Windows PowerShell 5.1, PowerShell 7.1.0

### Which operating system/environment are you running PnP PowerShell on?
(haven't tested anywhere else except...)
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by erwinvanhunen** [comment]
We're using a different JSON serialization engine in PnP.PowerShell (systext.text.json vs NewtonSoft.Json in the old one). We're aware that this works a bit different and I'm looking into making it easier.

To get the value out of it you can do: 

```powershell
$groups = Invoke-PnPSPRestMethod -Method get -Url "/_api/web/SiteUsers?`$filter=PrincipalType eq 4&`$select=Title,LoginName"
$groups.Values.GetRawText() | ConvertFrom-Json
```

**Comment by joshtransient** [comment]
Appreciate the backstory. Looking forward to seeing it work as well as it did with the legacy module.

**Comment by erwinvanhunen** [comment]
I just merged a change, ready for release in 1.2.7-nightly that will make it -much- easier to use the results. Please have a look, no more need to convert them to Json.

**Comment by joshtransient** [comment]
Looks great! Thanks again
